rtwcli/rtwdocker/rtwdocker/api/__init__.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2023, Stogl Robotics Consulting UG (haftungsbeschränkt)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import subprocess
+
+
+def start_container(container_instance_name: str):
+    if not container_instance_name:
+        print("No container name given. Can not start docker instance.")
+        return
+
+    print(f"Starting container instance: {container_instance_name}")
+    subprocess.run(["xhost", "+local:docker"])
+    subprocess.run(["docker", "start", container_instance_name])
```

Comment:
we should also fallback here to use existing rtw command. Then we move one by one - removing the old one and adding new one